src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,51 @@
+package racingcar.domain;
+
+public class Car {
+
+    private static final int START_POSITION = 0;
+    private static final int LIMIT_NAME_LENGTH = 5;
+    private static final int MOVABLE_OFFSET = 4;
+    private static final int FORWARD_AMOUNT = 1;
+
+    private final String name;
+    private final int position;
+    private final NumberGenerator numberGenerator;
+
+    public Car(final String name, final int position, final NumberGenerator numberGenerator) {
+        validateName(name);
+        this.name = name;
+        this.position = position;
+        this.numberGenerator = numberGenerator;
+    }
+
+    public static Car createDefault(final String name, final NumberGenerator numberGenerator) {
+        return new Car(name, START_POSITION, numberGenerator);
+    }
+
+    private void validateName(final String name) {
+        if (name == null || name.isEmpty() || name.length() > LIMIT_NAME_LENGTH) {
+            throw new IllegalArgumentException(
+                    String.format("자동차의 이름은 반드시 존재해야 하고, 최대 %d글자 이하여야 합니다.", LIMIT_NAME_LENGTH)
+            );
+        }
+    }
+
+    public Car moveForward() {
+        if (numberGenerator.generateNumber() >= MOVABLE_OFFSET) {
+            return new Car(getName(), getPosition() + FORWARD_AMOUNT, numberGenerator);
```

Comment:
```suggestion
            return new Car(name, position + FORWARD_AMOUNT, numberGenerator);
```
단순 필드정보를 가져오기위해서 getter를 쓸 필요는 없어보여요

Reply:
그렇네요..! 작성할때는 어떤 이유를 생각하고,, 작성했던것 같은데 지금 다시보니 굳이? 라는 생각이 듭니다!